.generator/conftest.py

```diff
@@ -252,6 +253,7 @@ def is_used(self, key):
     imports = defaultdict(set)
     given = defaultdict(dict)
 
+    unique_hash = "test-" + hashlib.sha256(unique.encode("utf-8")).hexdigest()[:16]
```

Comment:
I think I'd do something like that:

```
base64.b32encode(hashlib.sha256(unique.encode("utf-8")).digest())[:16].lower()
```

We should probably remove the prefix and add it in the tests instead.